ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -297,6 +298,24 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
         self.leadingView = leadingView
     }
 
+    /// - Parameters:
```

Comment:
Thoughts on tagging on to two of the existing `setup` functions and just adding an extra optional parameter (optional to make it not a breaking change)? I'd guess that would reduce binary size impact of this component and it definitely would make our interface simpler?